accessibility-checker-engine/help-v4/en-US/combobox_popup_reference.html

```diff
@@ -77,7 +73,7 @@ <h3 id="ruleMessage"></h3>
   </ul>
 ```
 
-Note, example code includes material copied from or derived from [W3C, ARIA Authoring Practices Guide (APG) ](https://www.w3.org/WAI/ARIA/apg/). Copyright © [2018-2021] W3C® (MIT, ERCIM, Keio, Beihang).
+Note: example code includes material copied from or derived from [W3C, ARIA Authoring Practices Guide (APG) ](https://www.w3.org/WAI/ARIA/apg/). Copyright © [2018-2021] W3C® (MIT, ERCIM, Keio, Beijing ).
```

Comment:
Beihang was correct.
```suggestion
Note: example code includes material copied from or derived from [W3C, ARIA Authoring Practices Guide (APG) ](https://www.w3.org/WAI/ARIA/apg/). Copyright © [2018-2021] W3C® (MIT, ERCIM, Keio, Beihang).
```